### PR TITLE
Refactor statsbeat utils

### PR DESCRIPTION
**Pull request by lzchen**

### CODE REVIEW

File: sdk/monitor/azure-monitor-opentelemetry-exporter/azure/monitor/opentelemetry/exporter/export/_base.py [hunk 1]
```diff
@@ -83,6 +82,7 @@ def __init__(self, **kwargs: Any) -> None:
         :keyword str storage_directory: Storage path in which to store retry files. Defaults to `<tempfile.gettempdir()>/opentelemetry-python-<your-instrumentation-key>`.
         :rtype: None
         """
+        self._connection_string = kwargs.get('connection_string')
```
Comment (jeremydvoss) [inline sdk/monitor/azure-monitor-opentelemetry-exporter/azure/monitor/opentelemetry/exporter/export/_base.py:85]: What's the purpose for this addition?

File: sdk/monitor/azure-monitor-opentelemetry-exporter/azure/monitor/opentelemetry/exporter/statsbeat/_utils.py [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+    ei_env = os.environ.get(_APPLICATIONINSIGHTS_STATS_LONG_EXPORT_INTERVAL_ENV_NAME)
+    if ei_env:
+        try:
+            return int(ei_env)
+        except ValueError:
+            return _DEFAULT_STATS_LONG_EXPORT_INTERVAL
+    return _DEFAULT_STATS_LONG_EXPORT_INTERVAL
+
+
+def _update_requests_map(type_name, value):
```
Comment (jeremydvoss) [inline sdk/monitor/azure-monitor-opentelemetry-exporter/azure/monitor/opentelemetry/exporter/statsbeat/_utils.py:56]: Is there a specific reason for making value required?

**Review by jeremydvoss — APPROVED** [review]

File: sdk/monitor/azure-monitor-opentelemetry-exporter/azure/monitor/opentelemetry/exporter/statsbeat/_utils.py (hunk 2 above)
Comment (lzchen) [reply sdk/monitor/azure-monitor-opentelemetry-exporter/azure/monitor/opentelemetry/exporter/statsbeat/_utils.py:56]: So we can consolidate the logic for when value is a number or a non-number.

File: sdk/monitor/azure-monitor-opentelemetry-exporter/azure/monitor/opentelemetry/exporter/export/_base.py (hunk 1 above)
Comment (lzchen) [reply sdk/monitor/azure-monitor-opentelemetry-exporter/azure/monitor/opentelemetry/exporter/export/_base.py:85]: Left in by accident.

**Pull request merged**